### ISSUE TITLE
Some corrections to what ends up in the outputted list

### ISSUE THREAD

**Issue by ShaunMaher** [ShaunMaher/USNJBackupQueue#7]
- We seem to be getting an extra entry that is /path/to/file.ext/refid
- We aren't getting the entire parent tree to the file, just the file and it's parent
- RelativePaths doesn't seem to be working (again)

```
E:/Data/BBAU/4 Finance & Accounts/4.12 Audit/2015 Dec/Australia/AA. EY Action List
E:/Data/BBAU/4 Finance & Accounts/4.12 Audit/2015 Dec/Australia/AA. EY Action List/Outstanding Items List -2016.04.18.xlsx
E:/Data/BBAU/4 Finance & Accounts/4.12 Audit/2015 Dec/Australia/AA. EY Action List/Outstanding Items List -2016.04.18.xlsx/99970
```

**Comment by ShaunMaher** [comment]
Also, WTF is this:

```
E:/Data/BBAU/7 Projects/1069 - Boomdell - 40m x DN400 Steel Casing/1.0 CONTRACT CORRESPONDENCE/SWMS-001 Horizontal Boring Rev 0 - Boomdell - 1069.docx/14. FX Exchange AED - USD.pdf
E:/Data/BBAU/7 Projects/1069 - Boomdell - 40m x DN400 Steel Casing/1.0 CONTRACT CORRESPONDENCE/SWMS-001 Horizontal Boring Rev 0 - Boomdell - 1069.docx/14. FX Exchange AED - USD.pdf/100642
E:/Data/BBAU/7 Projects/1069 - Boomdell - 40m x DN400 Steel Casing/1.0 CONTRACT CORRESPONDENCE/SWMS-001 Horizontal Boring Rev 0 - Boomdell - 1069.docx/14. FX Exchange USD - AED.pdf
E:/Data/BBAU/7 Projects/1069 - Boomdell - 40m x DN400 Steel Casing/1.0 CONTRACT CORRESPONDENCE/SWMS-001 Horizontal Boring Rev 0 - Boomdell - 1069.docx/14. FX Exchange USD - AED.pdf/100653
```

**Comment by ShaunMaher** [comment]
Issues resolved in f6e572c
